dropwizard-demo/test/com/test/ControllerTest.java

```diff
@@ -32,30 +32,6 @@ public void setUp() {
         tweetPost = Mockito.mock(Controller.class);
     }
 
-    @Test
-    public void testcase1_getTweets() {
```

Comment:
Y r u deleting test

Reply:
This testcases are related to duplicate code of my GET method. This  testcases is getting repeated , so I have deleted this testcases.